blog/_posts/2021-08-20-symbiosis-symbulation.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "Symbiosis in Symbulation"
+date: August 5, 2021
+author: Tiffany-Ellen Vo
+---
+
+# Symbiosis in Symbulation
+#### August 5, 2021 Tiffany-Ellen Vo
+- - - 
+
+[comment]: <> (How do I make it so the text is aligned with the picture?)
```

Comment:
So not positive on this, because I've not been able to run `jekyll serve` on my computer but some folks on the internet suggested putting two stars around text under a picture `*caption here*` to add a caption. But that might just be for the `![](hyperlink)` markdown method of adding pictures.